### PR TITLE
OSOE-685: Fix menu styling in Lombiq.BaseTheme

### PR DESCRIPTION
**Pull request by sarahelsaig**
[OSOE-685](https://lombiq.atlassian.net/browse/OSOE-685)

[OSOE-685]: https://lombiq.atlassian.net/browse/OSOE-685?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: test/Lombiq.OSOCE.Tests.UI/Tests/VisualVerificationTests/VisualVerificationTests.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Lombiq.OSOCE.Tests.UI.Tests.VisualVerificationTests;
+
+public class VisualVerificationTests : UITestBase
+{
+    public static readonly Size[] VisualVerificationSizes =
```
Comment (Psichorex) [inline test/Lombiq.OSOCE.Tests.UI/Tests/VisualVerificationTests/VisualVerificationTests.cs]: Will we use this anywhere else so that it can't be private?

**Review by Psichorex — CHANGES_REQUESTED** [review]

**Comment by Psichorex** [comment]
We have a conflict in Lombiq.GHA

**Comment by Psichorex** [comment]
Do you know anything about the failing Windows build? It can be just drunk and it might go away if we re-run.

**Comment by sarahelsaig** [comment]
It seems to be fine now.

**Pull request merged**